lib/ContentHelpers.php

```diff
@@ -11,6 +11,7 @@ public static function getCommonDetailFields(Entry $entry, $status, $locale)
     {
         return [
             'id' => $entry->id,
+            'entryType' => $entry->type->handle,
```

Comment:
I've added something similar [here](https://github.com/biglotteryfund/craft-dev/pull/107/files#diff-e4f83752c3ef565ee75acc378511be93R30) but passed the entire object so we can build strings etc. Might revisit this in light of the other PR.

Reply:
(just added this!)